### ISSUE TITLE
Scheduler improvement

### ISSUE THREAD

**Issue by yasker** [rancher/longhorn#192]
Time spent on deploying multiple VMs/volumes are very long. First VM started only after 10 minutes.

**Comment by cjellick** [comment]
Basically a duplicate of #144

**Comment by yasker** [comment]
@cjellick Yeah. Though #144 remains open only because I need to check what's happened to `empty volume` name.
